projects/media-viewer/src/lib/viewers/pdf-viewer/side-bar/side-bar.component.ts

```diff
@@ -76,17 +78,25 @@ export class SideBarComponent implements OnInit, OnChanges, OnDestroy {
   addBookmark(bookmark: Bookmark) {
     const documentId = this.extractDocumentId(this.url);
     const id = uuid();
-    bookmark.name = bookmark.name.substr(0,30);
-    this.bookmarksStore.dispatch(new CreateBookmark({ ...bookmark, documentId, id }));
-    this.toolbarButtons.sidebarOpen.next(true);
-    this.selectedView = 'bookmark'
```

Comment:
don't we want to show the bookmark once it's created? or are you doing that some other way

Reply:
Previously it was not actually showing the bookmark when it was created.. so I changed it so now it is